test/fixtures/v5/fork/left_exit.json

```diff
@@ -0,0 +1,22 @@
+{
+    "step": {
+        "maneuver": {
+            "type": "fork",
+            "modifier": "left"
+        },
+        "name": "Way Name",
+        "exits": "4A,4B"
+    },
+    "instructions": {
+        "de": "Links halten an der Gabelung auf Way Name",
+        "en": "Keep left at the fork onto Way Name",
```

Comment:
Is there a reason we don't add the exit information here? I can see how the "keep left" and the exit information might seem redundant but wouldn't it be a confirmation for the user to take a specific exit while keeping left?